python_ta/contracts/__init__.py

```diff
@@ -18,6 +18,7 @@
 import wrapt
 from typeguard import check_type
 
+ENABLE_CONTRACT_CHECKING = True
```

Comment:
Include a docstring below this variable, like `DEBUG_CONTRACTS`. That will be included when we generate docs for this module (you can check this by running `make docs`).

Reply:
@yibinglu I don't think you checked this on the HTML file properly. It turns out you need to add this to the `index.md` file to get it to appear. Please add an entry there as well.